src/sentry/static/sentry/app/stores/guideStore.jsx

```diff
@@ -48,7 +57,11 @@ const GuideStore = Reflux.createStore({
   onCloseGuideOrSupport() {
     let {currentGuide} = this.state;
     if (currentGuide) {
-      this.state.guidesSeen.add(currentGuide.id);
+      this.state.guides[
+        Object.keys(this.state.guides).find(key => {
```

Comment:
If you make the key the id you can simply do `this.state.guides[currentGuide.id].seen = true` Not a blocker though

Reply:
`currentGuide.id` isn't key, so that always returns null.

Reply:
that's why I mentioned "if you make the key the id" (in the guide definition)

Reply:
waiting to do this in another PR